iOS/src/toga_iOS/container.py

```diff
@@ -65,6 +71,14 @@ def __init__(self, content=None, layout_native=None, on_refresh=None):
         super().__init__(content=content, on_refresh=on_refresh)
         self.native = UIView.alloc().init()
         self.native.translatesAutoresizingMaskIntoConstraints = True
+        self.native.autoresizingMask = (
+            UIViewAutoresizingFlexibleLeftMargin
+            | UIViewAutoresizingFlexibleWidth
+            | UIViewAutoresizingFlexibleRightMargin
+            | UIViewAutoresizingFlexibleTopMargin
+            | UIViewAutoresizingFlexibleHeight
+            | UIViewAutoresizingFlexibleBottomMargin
```

Comment:
Have you got an example that demonstrates why Flexible(Top/Bottom/Left/Right)Margin is needed? Conceptually, it doesn't make sense to me that those should be required - the margins should always be 0 an fixed; and from my own testing, masking FlexibleWidth and FlexibleHeight seems to be sufficient.

Reply:
Yeah I didn't notice any issues with just FlexibleWidth and FlexibleHeight so I removed the other ones.